relation_engine_server/utils/json_validation.py

```diff
@@ -114,3 +135,32 @@ def _load_json_schema(file):
             return json.load(fd)
 
         raise TypeError('Unknown file type encountered: ' + file)
+
+
+class ExtendedRefResolver(RefResolver):
+
+    def resolve_remote(self, uri):
```

Comment:
This is almost entirely lifted directly from `jsonschema`

Reply:
Cool! 
Detecting & loading json or yaml is definitely nice